packages/common/test/directives/ng_optimized_image_spec.ts

```diff
@@ -720,6 +721,30 @@ describe('Image directive', () => {
         fixture.detectChanges();
       }).not.toThrowError(new RegExp('was updated after initialization'));
     });
+    it('should accept a safeUrl ngSrc value', () => {
+      @Component({
+        selector: 'test-cmp',
+        template: `<img
+              [ngSrc]="bypassImage"
+              width="400"
+              height="600"
+            >`
+      })
+      class TestComponent {
+        rawImage = `javascript:alert("Hi there")`;
+        bypassImage: SafeResourceUrl;
+        constructor(private sanitizer: DomSanitizer) {
+          this.bypassImage = sanitizer.bypassSecurityTrustResourceUrl(this.rawImage);
+        }
+      }
+      setupTestingModule({component: TestComponent});
+      const fixture = TestBed.createComponent(TestComponent);
+      fixture.detectChanges();
+
+      let nativeElement = fixture.nativeElement as HTMLElement;
+      let imgs = nativeElement.querySelectorAll('img')!;
```

Comment:
Nit: use regular queryselector if you just want the first item? Then you don't need to look at index 0.
```suggestion
      let img = nativeElement.querySelector('img')!;
```

Reply:
Good call, fixed